site/docs/guides/flowctl/troubleshoot-task.md

```diff
@@ -0,0 +1,61 @@
+---
+sidebar_position: 3
+---
+# Troubleshoot a task with flowctl
+
+flowctl offers the most advanced views of [task logs](../../concepts/advanced/logs-stats.md).
+If a task has errors or is failing in the web app, you'll be able to troubleshoot more effectively with flowctl.
+
+## Prerequisites
+
+To complete this workflow, you need:
+
+* An [Estuary account](../../getting-started/installation.md)
+
+* [flowctl installed locally](../../getting-started/installation.md#get-started-with-the-flow-cli)
+
+## Print task logs
+
+1. Authorize flowctl.
+
+   1. Go to the [CLI-API tab of the web app](https://dashboard.estuary.dev/admin/api) and copy your access token.
+
+   2. Run `flowctl auth token --token <paste-token-here>`
+
+2. Identify the name of the failing task in the web app; for example `myOrg/marketing/leads`.
+Use the tables on the Captures or Materializations pages of the web app to do so.
+
+3. Run `flowctl logs --task <task-name>`. You have several options to get more specific. For example:
+
+   * `flowctl logs --task myOrg/marketing/leads --follow` — If the task hasn't failed, continuously print logs as they're generated.
+
+   * `flowctl logs --task myOrg/marketing/leads --since 1h` — Print logs from approximately the last hour.
+   The actual output window is approximate and may somewhat exceed this time boundary.
```

Comment:
This may output logs from significantly _before_ the requested start time, but the requested start time will always be covered by the logs that were output. It kinda feels like we should try to clarify that :shrug: